components/list/test/list-item-placement-marker.visual-diff.js

```diff
@@ -0,0 +1,39 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('@brightspace-ui/visual-diff');
+
+describe('d2l-list-item-placement-marker', () => {
+
+	const visualDiff = new VisualDiff('list-item-placement-marker', __dirname);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await visualDiff.createPage(browser);
+		await page.goto(`${visualDiff.getBaseUrl()}/components/list/test/list-item-placement-marker.visual-diff.html`, {waitUntil: ['networkidle0', 'load']});
+		await page.bringToFront();
+	});
+
+	after(async() => await browser.close());
+
+	beforeEach(async() => await visualDiff.resetFocus(page));
+
+	describe('placement-marker', () => {
+		it('ltr', async function() {
+			const rect = await visualDiff.getRect(page, '#placement-marker');
+			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+		});
+		it('rtl', async function() {
+			const rect = await visualDiff.getRect(page, '#placement-marker-rtl');
+			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+		});
+		it('ltr-with-border', async function() {
```

Comment:
Would it be sufficient to just capture images with the border (i.e. only 2 screenshots)?  While the screenshots capture so much that a unit test can't, they definitely have a cost to them.  If the previous two tests capture something that these don't then it's perfectly fine, but would opt being conservative with the test time if possible.

Reply:
Sure sounds fine.  :)